nixos/modules/system/boot/systemd.nix

```diff
@@ -136,6 +139,16 @@ let
       "systemd-ask-password-wall.path"
       "systemd-ask-password-wall.service"
 
+      # Varlink APIs
+      "systemd-bootctl@.service"
+      "systemd-bootctl.socket"
+      "systemd-creds@.service"
+      "systemd-creds.socket"
+    ] ++ lib.optional cfg.package.withTpm2Tss [
+      "systemd-pcrlock@.service"
+      "systemd-pcrlock.socket"
```

Comment:
Main docs are https://www.freedesktop.org/software/systemd/man/latest/systemd-pcrlock.html

Source is [systemd-pcrlock@.service](https://raw.githubusercontent.com/systemd/systemd/main/units/systemd-pcrlock%40.service.in) and [systemd-pcrlock.socket](https://raw.githubusercontent.com/systemd/systemd/main/units/systemd-pcrlock.socket).